sdk/src/main/java/io/customer/sdk/api/HttpRequestRunner.kt

```diff
@@ -79,19 +82,24 @@ internal class HttpRequestRunnerImpl(
 
                 return Result.failure(CustomerIOError.Unauthorized())
             }
+            400 -> {
+                return Result.failure(CustomerIOError.BadRequest400())
+            }
             else -> {
                 var errorMessage = "No error body from API."
 
                 // by calling .string(), you are not able to get the error body again. retrofit clears the error body after calling .string()
                 response.errorBody()?.string()?.let { errorBodyString ->
-                    errorMessage = errorBodyString // if we can't parse the error body json, the raw response string is good to capture.
+                    errorMessage =
+                        errorBodyString // if we can't parse the error body json, the raw response string is good to capture.
 
                     parseCustomerIOErrorBody(errorBodyString)?.message?.let { parsedErrorMessage ->
                         errorMessage = parsedErrorMessage
                     }
                 }
 
-                val customerIOError = CustomerIOError.UnsuccessfulStatusCode(statusCode, errorMessage)
+                val customerIOError =
+                    CustomerIOError.UnsuccessfulStatusCode(statusCode, errorMessage)
```

Comment:
Not a blocker to this PR but I am curious. 

What is causing these format changes? ktlint? A Intellij plugin? 

It would be ideal if the formatting tools that were used in the project were the same across all engineers on the team. So if, for example, this change was caused by an intellij plugin, I think that plugin should be disabled or that rule gets applied to ktlint (or another tool that the team uses).